src/plugins/types.ts

```diff
@@ -33,4 +33,12 @@ export interface MonitoredPlugin {
     onSuccess(opts: OnSuccessOptions): void;
 
     onFailure(opts: OnFailureOptions): void;
+
+    increment(name: string, value: number, tags?: {[key: string]: string} | string[]): Promise<void>;
```

Comment:
Why `{[key: string]: string} | string[]`? Where does this type come from?

Reply:
Who knows....

Reply:
Who knows....